docs/resources/elb_l7policy.md

```diff
@@ -30,30 +101,111 @@ The following arguments are supported:
+  redirected. This parameter is mandatory when `action` is set to **FIXED_RESPONSE**. The `enhance_l7policy_enable` of
+  the listener must be set to **true**.
+  The [fixed_response_config](#fixed_response_config_object) structure is documented below.
+
+<a name="redirect_pools_extend_config_object"></a>
+The `redirect_pools_extend_config` block supports:
+
+* `rewrite_url_enabled` - (Optional, Bool) Whether the rewrite url is enabled.
+
+* `rewrite_url_config` - (Optional, List) The rewrite url config. This parameter is mandatory when `rewrite_url_enabled`
+  is set to **true**.
+  The [rewrite_url_config](#rewrite_url_config_object) structure is documented below.
+
+<a name="rewrite_url_config_object"></a>
+The `rewrite_url_config` block supports:
+
+* `host` - (Optional, String) The host name of the rewrite URL. The value can contain only letters, digits, and
+  periods (.) and must start with a letter or digit. Defaults to **${host}**, indicates inherit original value.
+
+* `path` - (Optional, String) The path of the rewrite URL. The value can contain only letters, digits, and special
+  characters _~';@^-%#&$.+?,=!:|\/(), and must start with a slash (/). Defaults to **${path}**, indicates inherit
+  original value.
+
+* `query` - (Optional, String) The query of the rewrite URL. The value can contain only letters, digits, and special
+  characters !$&'()+,-./:;=?@^_`. Defaults to **${query}**, indicates inherit original value.
+
+<a name="redirect_url_config_object"></a>
+The `redirect_url_config` block supports:
+
+* `status_code` - (Required, String) The status code returned after the requests are redirected.
+  Value options: **301**, **302**, **303**, **307**, **308**.
+
+* `protocol` - (Optional, String) The protocol for redirection. Value options: **HTTP**, **HTTPS**, **${protocol}**.
+  Defaults to **${protocol}**.
+
+* `host` - (Optional, String) The host name that requests are redirected to. The value can contain only letters,
+  digits, hyphens (-), and periods (.) and must start with a letter or digit. Defaults to **${host}**, indicating
+  that the host of the request will be used.
+
+* `port` - (Optional, String) The  port that requests are redirected to. Defaults to **${port}**, indicating that
+  the port of the request will be used.
+
+* `path` - (Optional, String) The path that requests are redirected to. The value can contain only letters, digits,
+  and special characters _~';@^- %#&$.*+?,=!:|/()[]{} and must start with a slash (/).
+  Defaults to **${path}**, indicating that the path of the request will be used.
+
+* `query` - (Optional, String) The query string set in the URL for redirection. The value is case-sensitive and can
+  contain only letters, digits, and special characters !$&'()*+,-./:;=?@^_`. Defaults to **${query}**, indicating that
+  the query string of the request will be used.
+  For example, in the [URL](https://www.xxx.com:8080/elb?type=loadbalancer), **${query}** indicates
```

Comment:
this URL is an example, not a real link, so better to find another for this

Reply:
fixed